Moki/client/src/js/dashboards/Microanalysis/MicroanalysisCharts.js

```diff
@@ -9,7 +9,7 @@ import ListChart from '../../charts/list_chart.js';
 import DonutChart from '../../charts/donut_chart.js';
 import LoadingScreenCharts from '../../helpers/LoadingScreenCharts';
 import ValueChart from '../../charts/value_chart.js';
-import { parseListData, parseIp, parseListDataCardinality, parseBucketData, parseAggDistinct, parseUri } from '@moki-client/es-response-parser';
+import { parseListData, parseIp, parseListDataCardinality, parseBucketData, parseAggDistinct, parseUri, parseListDataCardinalityDecrypt } from '@moki-client/es-response-parser';

```

Comment:
@Akaren13 how are we going to maintain public moki client vs. private moki client?
what is `parseListDataCardinalityDecrypt` going to do for the public moki client?